### PR TITLE
Fixed typos in Module 2

### PR DESCRIPTION
**Pull request by mghadam**
revised

### CODE REVIEW

File: lessons/02_spacetime/02_01_1DConvection.ipynb [hunk 1]
```diff
@@ -343,7 +343,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Remember that Python can also next commands, we could have instead written\n",
+    "Remember that we could have also combine the commands and write instead\n",
```
Comment (gforsyth) [inline lessons/02_spacetime/02_01_1DConvection.ipynb]: Can you change this to read:
"Remember that Python can also combine commands, we could have instead written..."

**Comment by gforsyth** [comment]
Thanks @mghadam!  Note that you don't need to open a new pull request each time, you can just make changes (either committing or rebasing) and push them to the same branch and the pull request will be updated

**Comment by mghadam** [comment]
done.

I opened the new pull request because I had moved my changes from master branch to a new one to do it in the right way.

**Comment by gforsyth** [comment]
Thanks, @mghadam !

**Pull request merged**